test/helper/Puppeteer_test.js

```diff
@@ -19,6 +14,48 @@ let page;
 let FS;
 const siteUrl = TestHelper.siteUrl();
 
+describe('Puppeteer - BasicAuth', () => {
+  before(() => {
+    global.codecept_dir = path.join(__dirname, '/../data');
+
+    I = new Puppeteer({
+      url: 'https://postman-echo.com',
```

Comment:
better to use local url

Reply:
@DavertMik but local url doesn't have basic auth?